commit 3249333a37bcb6725607f1058ea0394bacb265a3
Author: khoahvd <khoahvd@fsoft.com.vn>
Date:   Thu Nov 24 15:10:31 2016 +0700

    Add match of compatible r8a7742-iwg21m_q7 for pwm, sh and spi driver

diff --git a/drivers/pwm/pwm-renesas-tpu.c b/drivers/pwm/pwm-renesas-tpu.c
index aff6ba9..d534599 100644
--- a/drivers/pwm/pwm-renesas-tpu.c
+++ b/drivers/pwm/pwm-renesas-tpu.c
@@ -467,6 +467,7 @@ static int tpu_remove(struct platform_device *pdev)
 
 #ifdef CONFIG_OF
 static const struct of_device_id tpu_of_table[] = {
+    { .compatible = "renesas,tpu-r8a7742", },
 	{ .compatible = "renesas,tpu-r8a73a4", },
 	{ .compatible = "renesas,tpu-r8a7740", },
 	{ .compatible = "renesas,tpu-r8a7790", },
diff --git a/drivers/sh/pm_runtime.c b/drivers/sh/pm_runtime.c
index 4c376ac..b4c6729 100644
--- a/drivers/sh/pm_runtime.c
+++ b/drivers/sh/pm_runtime.c
@@ -91,6 +91,7 @@ static int __init sh_pm_runtime_init(void)
 		    !of_machine_is_compatible("renesas,r8a7792") &&
 		    !of_machine_is_compatible("renesas,r8a7793") &&
 		    !of_machine_is_compatible("renesas,r8a7794") &&
+           !of_machine_is_compatible("renesas,r8a7742") &&
 		    !of_machine_is_compatible("renesas,r8a7743") &&
 		    !of_machine_is_compatible("renesas,r8a7745") &&
 		    !of_machine_is_compatible("renesas,sh7372") &&

